### ISSUE TITLE
Configuration file

### ISSUE THREAD

**Issue by ggljzr** [ggljzr/trucksim-mqtt-telemetry#2]
Plugin should load its configuration from a JSON file. This file should contain stuff like

* MQTT broker host and port,
* which handlers should be registered.

**Comment by six-shooter** [comment]
I agree, I want to connect it to my Home Assistant MQTT, which I now cant :(

**Comment by ggljzr** [comment]
Ok noted, I will add this.

**Comment by ggljzr** [comment]
I have added a new release [0.2.1](https://github.com/ggljzr/trucksim-mqtt-telemetry/releases/tag/v0.2.1) that allows configuring the MQTT broker address via ``mqtt-config.json`` file.

When you start the game you should see a notification about "Request to use advanced SDK features" and then something like this in the game console (``~``):

<img width="603" height="121" alt="Image" src="https://github.com/user-attachments/assets/4eaa5f21-2d84-4627-9775-494d90ce5d13" />

Displayed IP address should be the one configured in ``mqtt-config.json``.

**Comment by ggljzr** [comment]
Added in dc34e42a34402f9bcb76088bb372d1b94bad285e.
